ios/wrappers/CioRctWrapper.swift

```diff
@@ -36,13 +36,11 @@ class CioRctWrapper: NSObject {
     
     @objc
     func identify(_ userId: String? = nil, traits: [String: Any]? = nil) {
-        if let userId, let traits {
-            CustomerIO.shared.identify(userId: userId, traits: traits)
-        } else if let userId {
-            CustomerIO.shared.identify(userId: userId, traits: traits)
-        } else {
+        guard let userId = userId else {
             // TODO: Add log when logger feature is implemented
+            return
         }
+        CustomerIO.shared.identify(userId: userId, traits: traits)
```

Comment:
Wouldn't this fail if `traits` are being attached to anonymous profile? Or do you plan to expose separate method for anonymous case?

Reply:
Do you mean if traits is `nil` ?

Reply:
No. I mean, when profile is anonymous i.e. `userId` is `nil`. But customer wants to attach `traits` to anonymous profile. Similar to [this method](https://github.com/customerio/customerio-ios/blob/b7d7d4fec2bd4be9f25b3b8937b01c7dc5360e0a/Sources/DataPipeline/CustomerIO%2BEvents.swift#L14) in iOS.

Reply:
Ah, you’re right. This doesn’t meet the anonymous profile criteria. Iwilll go ahead and update it.

Reply:
Makes sense, since the ticket will make any necessary changes required to handle the case for anonymous profile, I'm good with it 👍🏻